panda_QP_controller: # name of controller instance 
  type: linearmpc_panda/LinearMPCController
  arm_id: $(arg arm_id)
  joint_names:
      - $(arg arm_id)_joint1
      - $(arg arm_id)_joint2
      - $(arg arm_id)_joint3
      - $(arg arm_id)_joint4
      - $(arg arm_id)_joint5
      - $(arg arm_id)_joint6
      - $(arg arm_id)_joint7 